File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/reward_verify_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class RewardVerifyVO:
    """奖励验证VO"""
    created_at: str = None
    updated_at: str = None
    verified: bool = None
    reward_id: str = None
    reward_type: str = None
    reward_amount: int = None
    message: str = None
    verification_token: str = None

File: sdkwork-app-sdk-typescript/src/types/reward-verify-vo.ts
/** 奖励验证VO */
export interface RewardVerifyVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 是否验证通过 */
  verified?: boolean;
  /** 奖励ID */
  rewardId?: string;
  /** 奖励类型 */
  rewardType?: string;
  /** 奖励数量 */
  rewardAmount?: number;
  /** 消息 */
  message?: string;
  /** 验证令牌 */
  verificationToken?: string;
}
